src/index.js

```diff
@@ -61,10 +63,74 @@ const updateTask = () => {
     list[i].addEventListener('change', () => {
       collection.array[i].description = list[i].value;
       const stringData = JSON.stringify(collection.array);
-      localStorage.setItem('tasks', stringData);
+      localStorage.setItem('Tasks', stringData);
       window.location.reload();
     });
   }
 };
 updateTask();
-export default collection;
\ No newline at end of file
+
+// Add functionality to checkbox
+for (let i = 0; i < collection.array.length; i += 1) {
+  const checkbox = document.getElementById(`box${i}`);
+  checkbox.addEventListener('change', () => {
+    if (collection.array[i].complete === false) {
+      collection.array[i].complete = true;
+      localStorage.setItem('Tasks', JSON.stringify(collection.array));
+
+      const list = document.getElementById(`list${i}`);
+      list.style.textDecoration = 'line-through';
+    } else if (collection.array[i].complete === true) {
+      collection.array[i].complete = false;
+      localStorage.setItem('Tasks', JSON.stringify(collection.array));
+
+      const lists = document.getElementById(`list${i}`);
+      lists.style.textDecoration = 'none';
+    }
+  });
+}
+
+// Add functionality to clear button
+const clear = document.getElementById('clearBtn');
+clear.addEventListener('click', () => {
+  // filtering out incomplete tasks
+  const filtered = collection.array.filter((items) => items.complete === false);
+  const stringData = JSON.stringify(filtered);
+  for (let i = 0; i < collection.array.length; i += 1) {
+    const markedItem = document.getElementById(`item${i}`);
+    markedItem.remove();
+  }
+  // store filtered items into local storage
+  localStorage.setItem('Tasks', stringData);
+
+  const remove = document.querySelector('.trashIcon');
+  remove.style.display = 'none';
+
+  updateTask();
+  updateIndex();
+});
```

Comment:
- [ ] The clear button does not work as expected. Only completed tasks  (their completed value is true) should be deleted when the Clear all completed button is clicked.

Reply:
@medaminedev66 Thanks for the review. I couldn't figure out how to make the checkboxes stay the same before and after page refresh. could you please give me some insight on that? Thanks again!